proposals/forms.py

```diff
@@ -459,9 +459,9 @@ def __init__(self, *args, **kwargs):
         self.fields["parent"].queryset = self._get_parent_queryset()
 
     def _get_parent_queryset(self):
-        # Return all non-pre-assessments, that are not currently in review
+        # Return all proposals, that are not currently in review
         return (
-            Proposal.objects.filter(is_pre_assessment=False)
+            Proposal.objects.filter()
```

Comment:
Now that we're touching this, could you make this part of the `ProposalQuerySet` manager in `models.py`? What I want is that the manager provides the queryset of all revisable proposals, and this form only filters on the applicant part.

Reply:
What do you mean?